packages/php/blueprint/src/ResourceStorages/OrgThemeResourceStorage.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\ResourceStorages;
+
+class OrgThemeResourceStorage extends OrgPluginResourceStorage {
+	protected function get_download_link( $slug ) {
+		if ( ! function_exists( 'themes_api' ) ) {
+			include_once ABSPATH . '/wp-admin/includes/themes.php';
+		}
+		$info = \themes_api(
```

Comment:
Perhaps use `wp_themes_api`?

Reply:
Updated in [Use functions from trait](https://github.com/woocommerce/woocommerce/pull/49763/commits/c65ee6825520dce8fca5ddfa2e9152cb0ec765f6) (this also address other related comments)